schema/Query.graphql

```diff
@@ -28,4 +28,14 @@ type Query {
     Generate challenge to authenticate with WebAuthn
     """
     generateAuthenticatorChallenge(username: String!): AuthenticationWithWebPublicKeyCredential
+
+    """
+    Fetch Life document by its ID
+    """
+    getLife(id: String!): Life
```

Comment:
`id` here is not a `string` but an `ObjectId`

Reply:
Noted! I have changed all the files to reflect the proper type `ObjectID` as defined in the GraphQL scalars.